shared/django_apps/reports/models.py

```diff
@@ -257,6 +257,9 @@ class Test(models.Model):
     # for example: the same test being run on windows vs. mac
     flags_hash = models.TextField()
 
+    failure_rate = models.FloatField(null=True)
+    commits_where_fail = ArrayField(models.TextField(), null=True)
```

Comment:
I hope the application code will enforce a max length on this. We have had issues in the past with array / json objects in the database that grow without a limit.

Reply:
This is more of a rollup field and  will be capped at commits that failed in the last 30 days, so it shouldn't grow unbounded forever.